package.json

```diff
@@ -123,4 +123,4 @@
       "prettier --write"
     ]
   }
-}
+}
```

Comment:
Your editor is removing last new line, please disable this)

Reply:
fixed my vscode settings)